frontend/webpack.config.js

```diff
@@ -9,7 +9,10 @@ const API_URL = {
   development: 'http://localhost:3000',
 };
 
+const ASSET_PATH = process.env.ASSET_PATH || '/';
+
 module.exports = {
+  entry: './src/index.jsx',
```

Comment:
I think this is not required as `HtmlWebpackPlugin` would handle injected of the `main.js` script file

Reply:
have updated @gondar00